src/main/java/be/seeseemelk/mockbukkit/entity/PillagerMock.java

```diff
@@ -0,0 +1,46 @@
+package be.seeseemelk.mockbukkit.entity;
+
+import be.seeseemelk.mockbukkit.ServerMock;
+import be.seeseemelk.mockbukkit.inventory.InventoryMock;
+import org.bukkit.Sound;
+import org.bukkit.entity.Pillager;
+import org.bukkit.event.inventory.InventoryType;
+import org.bukkit.inventory.Inventory;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.UUID;
+
+/**
+ * Mock implementation of a {@link Pillager}.
+ *
+ * @see IllagerMock
+ */
+public class PillagerMock extends IllagerMock implements Pillager, MockRangedEntity<PillagerMock>
+{
+
+	private final Inventory inventory = new InventoryMock(this, 5, InventoryType.CHEST);
```

Comment:
I don't know that much about inventories, but from looking into the paper source code I found that the main hand should have a crossbow item in it for pillagers.

Reply:
Good catch! I totally forgot about the Crossbow.
PR has been updated with it.